tests/Shared/XUnitLoggerTests.cs

```diff
@@ -659,6 +659,41 @@ public static void XUnitLogger_Log_Logs_Message_If_Scopes_Included_And_There_Is_
         outputHelper.Received(1).WriteLine(expected);
     }
 
+#if NET8_0_OR_GREATER
+
+    [Fact]
+    public static void XUnitLogger_Log_Logs_Message_With_TimeProvider()
+    {
+        // Arrange
+        var outputHelper = Substitute.For<ITestOutputHelper>();
+        string name = "MyName";
+
+        var timeProvider = Substitute.For<TimeProvider>();
+        timeProvider.LocalTimeZone.Returns(TimeZoneInfo.Utc);
+        timeProvider.GetUtcNow().Returns(new DateTimeOffset(2019, 05, 12, 13, 47, 41, 123, 456, TimeSpan.Zero));
```

Comment:
One last change and then I think this is all done and good to go.

Please change the time zone to whatever maps to `Europe/London` (which should be `+01:00` on the date on the test, because it's in late Spring).

That will then validate that local time was definitely used, as the test as-is doesn't definitively _prove_ that as the local time is the same as the UTC time.

You can see the same concept here. The local time is 17:12 in August in the summer:

https://github.com/martincostello/xunit-logging/blob/de2b654fe80a5e4145556babbae542a8e6f435d3/tests/Shared/XUnitLoggerTests.cs#L697

But the time in the log message is an hour earlier as it's UTC:

https://github.com/martincostello/xunit-logging/blob/de2b654fe80a5e4145556babbae542a8e6f435d3/tests/Shared/XUnitLoggerTests.cs#L322

Reply:
The change I was getting at (without having a debugger/compiler to hand) was the latter one to change the local time zone.

I don't think there's a need for a data-driven test, as I'm fairly sure the existing tests already cover that.